make.sh

```diff
@@ -24,40 +24,40 @@ function build_iOS {
 
 # build Android lib for only one supported architecture
 function build_android {
-    if [ -z "$NDK" ]; then
-        echo "ERROR! Please set \$NDK to point at your Android NDK directory."
-        exit 1
-    fi
-    HOSTOS=$(uname -s | tr 'LD' 'ld')
-    HOSTARCH=$(uname -m)
-
-    TARGARCH="$1"
-    shift
-
-    case "$TARGARCH" in
-      arm)
-          [ -n "$APILEVEL" ] || APILEVEL="android-14"  # default to ICS
-          [ -n "$GCCVER" ] || GCCVER="4.8"
-          CROSS=arm-linux-androideabi-
-          ;;
-      arm64)
-          [ -n "$APILEVEL" ] || APILEVEL="android-21"  # first with arm64
-          [ -n "$GCCVER" ] || GCCVER="4.9"
-          CROSS=aarch64-linux-android-
-          ;;
-
-      *)
-          echo "ERROR! Building for Android on $1 is not currently supported."
-          exit 1
-          ;;
-    esac
-
-    TOOLCHAIN="$NDK/toolchains/$CROSS$GCCVER/prebuilt/$HOSTOS-$HOSTARCH"
-    PLATFORM="$NDK/platforms/$APILEVEL/arch-$TARGARCH"
+	if [ -z "$NDK" ]; then
+		echo "ERROR! Please set \$NDK to point at your Android NDK directory."
+		exit 1
+	fi
+	HOSTOS="$(uname -s | tr LD ld)"
```

Comment:
i assume because of Linux and Darwin.. but thats a weird assumption when we can just lowercase the whole thing